src/Tasks/Microsoft.NET.Build.Tasks/ResolvePackageAssets.cs

```diff
@@ -1038,6 +1037,12 @@ private void WriteRuntimeAssemblies()
 
             private void WriteRuntimeTargets()
             {
+                // Only write runtime targets if the TFM is netcoreapp
```

Comment:
I wonder if we should still return them, but not add them to the copy/publish list. Just in case someone out there has done something funky and has there own custom loader to load them, they can take the returned items from here and slap them in themselves. I might be being too paranoid here.

Reply:
Probably worth a comment that other runtimes can't load these dynamically.

Reply:
I don't have that strong of an opinion on this, so I can be convinced to fix it in the targets rather than the task.

However, I highly doubt anyone would rely on this behavior since it only affected publishing .NET Framework-targeted non-exe projects prior to 3.0.

Reply:
Yeah, this is cleaner. Letr's go with it. Wouldn't be hard to add something back if someone screams and I doubt it.

Reply:
I'll improve the comment then.